[2025-01-26 00:26:33,292: INFO: main: welcome to our datascience project: ]
[2025-01-26 00:27:15,756: INFO: main: welcome to our datascience project: ]
[2025-01-26 00:27:46,604: INFO: main: welcome to our datascience project: ]
[2025-01-26 00:27:46,605: WARNING: main: red light: ]
[2025-01-26 00:28:20,601: INFO: main: welcome to our datascience project: ]
[2025-01-26 00:28:20,602: WARNING: main: red light: ]
[2025-01-26 00:28:20,602: ERROR: main: value missing!: ]
[2025-01-26 00:29:25,200: INFO: main: welcome to our datascience project: ]
[2025-01-26 00:29:25,200: WARNING: main: red light: ]
[2025-01-26 00:29:25,200: ERROR: main: value missing!: ]
[2025-01-26 00:29:25,200: CRITICAL: main: this is serious: ]
[2025-01-27 23:43:03,258: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-27 23:43:03,383: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-27 23:48:46,161: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-27 23:48:46,166: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-27 23:48:46,172: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-27 23:48:46,303: INFO: common: created directory at: artifacts: ]
[2025-01-27 23:48:46,309: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-27 23:49:27,829: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-27 23:49:27,832: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-27 23:49:27,836: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-27 23:49:27,841: INFO: common: created directory at: artifacts: ]
[2025-01-27 23:49:27,844: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-27 23:50:12,112: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-27 23:50:12,116: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-27 23:50:12,122: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-27 23:50:12,126: INFO: common: created directory at: artifacts: ]
[2025-01-27 23:50:12,128: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-27 23:50:16,112: INFO: 2447044660: artifacts/data_ingestion/data.zip download! with following info: 
Server: GitHub.com
Date: Mon, 27 Jan 2025 18:20:12 GMT
Content-Type: text/html; charset=utf-8
Vary: X-PJAX, X-PJAX-Container, Turbo-Visit, Turbo-Frame, Accept-Encoding, Accept, X-Requested-With
ETag: W/"3d9e2a2d96646214255a7029ab9913e8"
Cache-Control: max-age=0, private, must-revalidate
Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
X-Frame-Options: deny
X-Content-Type-Options: nosniff
X-XSS-Protection: 0
Referrer-Policy: no-referrer-when-downgrade
Content-Security-Policy: default-src 'none'; base-uri 'self'; child-src github.com/assets-cdn/worker/ github.com/webpack/ github.com/assets/ gist.github.com/assets-cdn/worker/; connect-src 'self' uploads.github.com www.githubstatus.com collector.github.com raw.githubusercontent.com api.github.com github-cloud.s3.amazonaws.com github-production-repository-file-5c1aeb.s3.amazonaws.com github-production-upload-manifest-file-7fdce7.s3.amazonaws.com github-production-user-asset-6210df.s3.amazonaws.com *.rel.tunnels.api.visualstudio.com wss://*.rel.tunnels.api.visualstudio.com objects-origin.githubusercontent.com copilot-proxy.githubusercontent.com proxy.individual.githubcopilot.com proxy.business.githubcopilot.com proxy.enterprise.githubcopilot.com *.actions.githubusercontent.com wss://*.actions.githubusercontent.com productionresultssa0.blob.core.windows.net/ productionresultssa1.blob.core.windows.net/ productionresultssa2.blob.core.windows.net/ productionresultssa3.blob.core.windows.net/ productionresultssa4.blob.core.windows.net/ productionresultssa5.blob.core.windows.net/ productionresultssa6.blob.core.windows.net/ productionresultssa7.blob.core.windows.net/ productionresultssa8.blob.core.windows.net/ productionresultssa9.blob.core.windows.net/ productionresultssa10.blob.core.windows.net/ productionresultssa11.blob.core.windows.net/ productionresultssa12.blob.core.windows.net/ productionresultssa13.blob.core.windows.net/ productionresultssa14.blob.core.windows.net/ productionresultssa15.blob.core.windows.net/ productionresultssa16.blob.core.windows.net/ productionresultssa17.blob.core.windows.net/ productionresultssa18.blob.core.windows.net/ productionresultssa19.blob.core.windows.net/ github-production-repository-image-32fea6.s3.amazonaws.com github-production-release-asset-2e65be.s3.amazonaws.com insights.github.com wss://alive.github.com api.githubcopilot.com api.individual.githubcopilot.com api.business.githubcopilot.com api.enterprise.githubcopilot.com; font-src github.githubassets.com; form-action 'self' github.com gist.github.com copilot-workspace.githubnext.com objects-origin.githubusercontent.com; frame-ancestors 'none'; frame-src viewscreen.githubusercontent.com notebooks.githubusercontent.com; img-src 'self' data: blob: github.githubassets.com media.githubusercontent.com camo.githubusercontent.com identicons.github.com avatars.githubusercontent.com private-avatars.githubusercontent.com github-cloud.s3.amazonaws.com objects.githubusercontent.com secured-user-images.githubusercontent.com/ user-images.githubusercontent.com/ private-user-images.githubusercontent.com opengraph.githubassets.com github-production-user-asset-6210df.s3.amazonaws.com customer-stories-feed.github.com spotlights-feed.github.com objects-origin.githubusercontent.com *.githubusercontent.com; manifest-src 'self'; media-src github.com user-images.githubusercontent.com/ secured-user-images.githubusercontent.com/ private-user-images.githubusercontent.com github-production-user-asset-6210df.s3.amazonaws.com gist.github.com; script-src github.githubassets.com; style-src 'unsafe-inline' github.githubassets.com; upgrade-insecure-requests; worker-src github.com/assets-cdn/worker/ github.com/webpack/ github.com/assets/ gist.github.com/assets-cdn/worker/
Accept-Ranges: bytes
Set-Cookie: _gh_sess=luzbmKj3le4dbgyqgYQCsNYJ46Rt2SpKMQYGAnoqJgGhbMKGSo6zcVR3Qz%2FOlTcADhCMY3lAsfcEkdnReyZgDeSacKBHQ8tVYrbZ4Uy%2Bju2RUNk%2FDxtTB%2FwTDxajgmMp7G5tXYRmM80ZFmDMlN011Q29eXCDdlFzUu%2BL7Q%2F644G2S%2FXY9lRfJYyXy4UoNsaOVq0cUL3%2F5wYC1MXgsfPipb0ZQIRSDUEkdmHxFEEOBgWRg3%2Fx%2BmVWHw4LkMDir%2FV3YDjZb8wFRAXw%2FC9wb1VqsQ%3D%3D--L2fRraOqTnic8X7l--HEZBmbtUr9ii1iO1jEDmYA%3D%3D; Path=/; HttpOnly; Secure; SameSite=Lax
Set-Cookie: _octo=GH1.1.1212719110.1738002011; Path=/; Domain=github.com; Expires=Tue, 27 Jan 2026 18:20:11 GMT; Secure; SameSite=Lax
Set-Cookie: logged_in=no; Path=/; Domain=github.com; Expires=Tue, 27 Jan 2026 18:20:11 GMT; HttpOnly; Secure; SameSite=Lax
X-GitHub-Request-Id: E015:276D2:1C569E:2038B2:6797CE5B
connection: close
Transfer-Encoding: chunked

: ]
[2025-01-27 23:51:40,356: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-27 23:51:40,442: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-27 23:51:40,445: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-27 23:51:40,447: INFO: common: created directory at: artifacts: ]
[2025-01-27 23:51:40,449: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-27 23:51:40,451: INFO: 2447044660: File already exists: ]
[2025-01-27 23:55:17,439: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-27 23:55:17,445: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-27 23:55:17,448: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-27 23:55:17,451: INFO: common: created directory at: artifacts: ]
[2025-01-27 23:55:17,454: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-27 23:55:18,659: INFO: 2447044660: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 23329
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "c69888a4ae59bc5a893392785a938ccd4937981c06ba8a9d6a21aa52b4ab5b6e"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 42D7:0885:1E72C:293D0:6797CF8D
Accept-Ranges: bytes
Date: Mon, 27 Jan 2025 18:25:17 GMT
Via: 1.1 varnish
X-Served-By: cache-del21733-DEL
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1738002317.416686,VS0,VE505
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: abe16e17c3262f2cb29bc20e8ea943e197f5873e
Expires: Mon, 27 Jan 2025 18:30:17 GMT
Source-Age: 0

: ]
[2025-01-28 00:21:16,515: INFO: main: >>>>>> stage Data ingestion Stage started <<<<<<: ]
[2025-01-28 00:21:16,527: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 00:21:16,540: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 00:21:16,542: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 00:21:16,543: INFO: common: created directory at: artifacts: ]
[2025-01-28 00:21:16,544: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-28 00:21:17,843: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 23329
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "c69888a4ae59bc5a893392785a938ccd4937981c06ba8a9d6a21aa52b4ab5b6e"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 42D7:0885:1E72C:293D0:6797CF8D
Accept-Ranges: bytes
Date: Mon, 27 Jan 2025 18:51:17 GMT
Via: 1.1 varnish
X-Served-By: cache-del21727-DEL
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1738003877.816044,VS0,VE313
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 705e291126c44cb47dc8ec730b563ba03e848931
Expires: Mon, 27 Jan 2025 18:56:17 GMT
Source-Age: 0

: ]
[2025-01-28 00:21:17,875: INFO: main: >>>>>> stage Data ingestion Stage completed <<<<<<

x==========x: ]
[2025-01-28 21:39:30,535: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 21:39:30,653: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 21:39:30,675: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 21:39:30,701: INFO: common: created directory at: artifacts: ]
[2025-01-28 21:39:30,718: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-28 21:39:54,007: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 21:39:54,013: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 21:39:54,019: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 21:39:54,023: INFO: common: created directory at: artifacts: ]
[2025-01-28 21:39:54,025: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-28 21:43:43,519: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 21:43:43,526: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 21:43:43,532: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 21:43:43,535: INFO: common: created directory at: artifacts: ]
[2025-01-28 21:43:43,538: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-28 21:44:17,329: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 21:44:17,335: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 21:44:17,343: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 21:44:17,346: INFO: common: created directory at: artifacts: ]
[2025-01-28 21:44:17,349: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-28 21:50:05,804: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 21:50:05,810: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 21:50:05,816: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 21:50:05,818: INFO: common: created directory at: artifacts: ]
[2025-01-28 21:50:05,820: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-28 21:50:39,373: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 21:50:39,378: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 21:50:39,385: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 21:50:39,388: INFO: common: created directory at: artifacts: ]
[2025-01-28 21:50:39,392: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-28 21:50:44,213: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 21:50:44,219: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 21:50:44,225: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 21:50:44,230: INFO: common: created directory at: artifacts: ]
[2025-01-28 21:50:44,232: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-28 21:59:10,029: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 21:59:10,097: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 21:59:10,109: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 21:59:10,134: INFO: common: created directory at: artifacts: ]
[2025-01-28 21:59:10,140: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-28 21:59:31,573: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 21:59:31,578: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 21:59:31,585: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 21:59:31,589: INFO: common: created directory at: artifacts: ]
[2025-01-28 21:59:31,592: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-28 22:00:21,665: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 22:00:21,670: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 22:00:21,676: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 22:00:21,679: INFO: common: created directory at: artifacts: ]
[2025-01-28 22:00:21,681: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-28 22:01:19,993: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 22:01:19,997: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 22:01:20,012: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 22:01:20,014: INFO: common: created directory at: artifacts: ]
[2025-01-28 22:01:20,017: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-28 22:01:59,221: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 22:01:59,226: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 22:01:59,232: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 22:01:59,234: INFO: common: created directory at: artifacts: ]
[2025-01-28 22:01:59,236: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-28 22:02:13,983: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 22:02:13,986: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 22:02:13,992: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 22:02:13,998: INFO: common: created directory at: artifacts: ]
[2025-01-28 22:02:14,000: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-28 22:02:57,643: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 22:02:57,647: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 22:02:57,655: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 22:02:57,658: INFO: common: created directory at: artifacts: ]
[2025-01-28 22:02:57,661: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-28 22:03:57,081: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 22:03:57,084: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 22:03:57,090: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 22:03:57,093: INFO: common: created directory at: artifacts: ]
[2025-01-28 22:03:57,095: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-28 22:04:49,860: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 22:04:49,865: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 22:04:49,870: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 22:04:49,872: INFO: common: created directory at: artifacts: ]
[2025-01-28 22:04:49,875: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-28 22:05:58,461: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 22:05:58,467: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 22:05:58,472: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 22:05:58,474: INFO: common: created directory at: artifacts: ]
[2025-01-28 22:05:58,477: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-28 22:22:05,352: INFO: main: >>>>>> stage Data Validation stage started <<<<<<: ]
[2025-01-28 22:22:05,356: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 22:22:05,357: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 22:22:05,361: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 22:22:05,362: INFO: common: created directory at: artifacts: ]
[2025-01-28 22:22:05,363: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-28 22:22:05,363: INFO: data_ingestion: File already exists: ]
[2025-01-28 22:22:05,476: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x: ]
[2025-01-28 22:22:05,477: INFO: main: >>>>>> stage Data Validation stage started <<<<<<: ]
[2025-01-28 22:22:05,484: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 22:22:05,486: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 22:22:05,493: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 22:22:05,494: INFO: common: created directory at: artifacts: ]
[2025-01-28 22:22:05,495: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-28 22:22:05,534: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x: ]
[2025-01-28 23:05:36,094: INFO: main: >>>>>> stage Data Trnasformation Stage started <<<<<<: ]
[2025-01-28 23:05:36,115: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 23:05:36,162: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 23:05:36,184: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 23:05:36,194: INFO: common: created directory at: artifacts: ]
[2025-01-28 23:05:36,195: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-28 23:05:36,197: INFO: data_ingestion: File already exists: ]
[2025-01-28 23:05:36,259: INFO: main: >>>>>> stage Data Trnasformation Stage completed <<<<<<

x==========x: ]
[2025-01-28 23:05:36,260: INFO: main: >>>>>> stage Data Validation stage started <<<<<<: ]
[2025-01-28 23:05:36,265: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 23:05:36,266: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 23:05:36,269: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 23:05:36,270: INFO: common: created directory at: artifacts: ]
[2025-01-28 23:05:36,270: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-28 23:05:36,447: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x: ]
[2025-01-28 23:05:36,449: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<: ]
[2025-01-28 23:05:36,467: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x: ]
[2025-01-28 23:08:45,217: INFO: main: >>>>>> stage Data Trnasformation Stage started <<<<<<: ]
[2025-01-28 23:08:45,220: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 23:08:45,221: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 23:08:45,224: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 23:08:45,226: INFO: common: created directory at: artifacts: ]
[2025-01-28 23:08:45,228: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-28 23:08:45,231: INFO: data_ingestion: File already exists: ]
[2025-01-28 23:08:45,237: INFO: main: >>>>>> stage Data Trnasformation Stage completed <<<<<<

x==========x: ]
[2025-01-28 23:08:45,241: INFO: main: >>>>>> stage Data Validation stage started <<<<<<: ]
[2025-01-28 23:08:45,245: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 23:08:45,246: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 23:08:45,250: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 23:08:45,251: INFO: common: created directory at: artifacts: ]
[2025-01-28 23:08:45,251: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-28 23:08:45,281: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x: ]
[2025-01-28 23:08:45,283: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<: ]
[2025-01-28 23:08:45,294: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x: ]
[2025-01-28 23:09:36,149: INFO: main: >>>>>> stage Data Trnasformation Stage started <<<<<<: ]
[2025-01-28 23:09:36,157: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 23:09:36,160: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 23:09:36,169: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 23:09:36,170: INFO: common: created directory at: artifacts: ]
[2025-01-28 23:09:36,171: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-28 23:09:36,172: INFO: data_ingestion: File already exists: ]
[2025-01-28 23:09:36,177: INFO: main: >>>>>> stage Data Trnasformation Stage completed <<<<<<

x==========x: ]
[2025-01-28 23:09:36,185: INFO: main: >>>>>> stage Data Validation stage started <<<<<<: ]
[2025-01-28 23:09:36,214: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 23:09:36,220: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 23:09:36,232: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 23:09:36,235: INFO: common: created directory at: artifacts: ]
[2025-01-28 23:09:36,236: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-28 23:09:36,306: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x: ]
[2025-01-28 23:09:36,307: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<: ]
[2025-01-28 23:09:36,310: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x: ]
[2025-01-28 23:10:15,689: INFO: main: >>>>>> stage Data Trnasformation Stage started <<<<<<: ]
[2025-01-28 23:10:15,691: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 23:10:15,692: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 23:10:15,695: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 23:10:15,695: INFO: common: created directory at: artifacts: ]
[2025-01-28 23:10:15,696: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-28 23:10:15,696: INFO: data_ingestion: File already exists: ]
[2025-01-28 23:10:15,699: INFO: main: >>>>>> stage Data Trnasformation Stage completed <<<<<<

x==========x: ]
[2025-01-28 23:10:15,699: INFO: main: >>>>>> stage Data Validation stage started <<<<<<: ]
[2025-01-28 23:10:15,702: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 23:10:15,704: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 23:10:15,708: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 23:10:15,709: INFO: common: created directory at: artifacts: ]
[2025-01-28 23:10:15,710: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-28 23:10:15,743: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x: ]
[2025-01-28 23:10:15,744: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<: ]
[2025-01-28 23:10:15,758: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x: ]
[2025-01-28 23:11:04,898: INFO: main: >>>>>> stage Data Trnasformation Stage started <<<<<<: ]
[2025-01-28 23:11:04,901: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 23:11:04,901: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 23:11:04,905: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 23:11:04,905: INFO: common: created directory at: artifacts: ]
[2025-01-28 23:11:04,908: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-28 23:11:04,909: INFO: data_ingestion: File already exists: ]
[2025-01-28 23:11:04,916: INFO: main: >>>>>> stage Data Trnasformation Stage completed <<<<<<

x==========x: ]
[2025-01-28 23:11:04,917: INFO: main: >>>>>> stage Data Validation stage started <<<<<<: ]
[2025-01-28 23:11:04,921: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 23:11:04,922: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 23:11:04,930: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 23:11:04,932: INFO: common: created directory at: artifacts: ]
[2025-01-28 23:11:04,934: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-28 23:11:04,972: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x: ]
[2025-01-28 23:11:04,979: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<: ]
[2025-01-28 23:11:04,986: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x: ]
[2025-01-28 23:12:44,578: INFO: main: >>>>>> stage Data Trnasformation Stage started <<<<<<: ]
[2025-01-28 23:12:44,581: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 23:12:44,581: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 23:12:44,585: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 23:12:44,585: INFO: common: created directory at: artifacts: ]
[2025-01-28 23:12:44,586: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-28 23:12:44,587: INFO: data_ingestion: File already exists: ]
[2025-01-28 23:12:44,591: INFO: main: >>>>>> stage Data Trnasformation Stage completed <<<<<<

x==========x: ]
[2025-01-28 23:12:44,592: INFO: main: >>>>>> stage Data Validation stage started <<<<<<: ]
[2025-01-28 23:12:44,597: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 23:12:44,601: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 23:12:44,604: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 23:12:44,606: INFO: common: created directory at: artifacts: ]
[2025-01-28 23:12:44,606: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-28 23:12:44,634: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x: ]
[2025-01-28 23:12:44,638: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<: ]
[2025-01-28 23:12:44,644: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 23:12:44,645: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 23:12:44,649: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 23:12:44,650: INFO: common: created directory at: artifacts: ]
[2025-01-28 23:12:44,653: INFO: common: created directory at: artifacts/data_transformation: ]
[2025-01-28 23:12:44,653: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x: ]
[2025-01-28 23:13:46,194: INFO: main: >>>>>> stage Data Trnasformation Stage started <<<<<<: ]
[2025-01-28 23:13:46,211: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 23:13:46,217: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 23:13:46,224: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 23:13:46,226: INFO: common: created directory at: artifacts: ]
[2025-01-28 23:13:46,233: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-28 23:13:46,235: INFO: data_ingestion: File already exists: ]
[2025-01-28 23:13:46,240: INFO: main: >>>>>> stage Data Trnasformation Stage completed <<<<<<

x==========x: ]
[2025-01-28 23:13:46,242: INFO: main: >>>>>> stage Data Validation stage started <<<<<<: ]
[2025-01-28 23:13:46,256: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 23:13:46,259: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 23:13:46,272: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 23:13:46,274: INFO: common: created directory at: artifacts: ]
[2025-01-28 23:13:46,278: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-28 23:13:46,312: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x: ]
[2025-01-28 23:13:46,313: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<: ]
[2025-01-28 23:13:46,321: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-28 23:13:46,325: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-28 23:13:46,330: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-28 23:13:46,331: INFO: common: created directory at: artifacts: ]
[2025-01-28 23:13:46,332: INFO: common: created directory at: artifacts/data_transformation: ]
[2025-01-28 23:13:46,549: INFO: data_transformation: Splited data into training and test sets: ]
[2025-01-28 23:13:46,549: INFO: data_transformation: (1199, 12): ]
[2025-01-28 23:13:46,550: INFO: data_transformation: (400, 12): ]
[2025-01-28 23:13:46,564: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x: ]
[2025-01-29 14:27:34,261: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<: ]
[2025-01-29 14:27:34,282: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:27:34,364: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:27:34,367: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:27:34,396: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:27:34,400: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-29 14:27:34,401: INFO: data_ingestion: File already exists: ]
[2025-01-29 14:27:34,475: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x: ]
[2025-01-29 14:27:34,476: INFO: main: >>>>>> stage Data Validation stage started <<<<<<: ]
[2025-01-29 14:27:34,481: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:27:34,482: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:27:34,486: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:27:34,488: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:27:34,489: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-29 14:27:34,603: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x: ]
[2025-01-29 14:27:34,608: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<: ]
[2025-01-29 14:27:34,624: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:27:34,625: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:27:34,629: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:27:34,631: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:27:34,632: INFO: common: created directory at: artifacts/data_transformation: ]
[2025-01-29 14:27:34,790: INFO: data_transformation: Splited data into training and test sets: ]
[2025-01-29 14:27:34,791: INFO: data_transformation: (1199, 12): ]
[2025-01-29 14:27:34,791: INFO: data_transformation: (400, 12): ]
[2025-01-29 14:27:34,797: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x: ]
[2025-01-29 14:27:34,798: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<: ]
[2025-01-29 14:27:34,799: ERROR: main: 'ModelTrainerTrainingPipeline' object has no attribute 'initiate_model_training': ]
Traceback (most recent call last):
  File "D:\DS\project\datascienceproject\main.py", line 40, in <module>
    data_ingestion.initiate_model_training()
AttributeError: 'ModelTrainerTrainingPipeline' object has no attribute 'initiate_model_training'
[2025-01-29 14:30:02,512: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<: ]
[2025-01-29 14:30:02,516: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:30:02,519: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:30:02,521: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:30:02,522: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:30:02,523: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-29 14:30:02,523: INFO: data_ingestion: File already exists: ]
[2025-01-29 14:30:02,528: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x: ]
[2025-01-29 14:30:02,529: INFO: main: >>>>>> stage Data Validation stage started <<<<<<: ]
[2025-01-29 14:30:02,536: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:30:02,539: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:30:02,543: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:30:02,545: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:30:02,545: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-29 14:30:02,575: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x: ]
[2025-01-29 14:30:02,577: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<: ]
[2025-01-29 14:30:02,585: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:30:02,587: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:30:02,590: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:30:02,591: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:30:02,593: INFO: common: created directory at: artifacts/data_transformation: ]
[2025-01-29 14:30:02,632: INFO: data_transformation: Splited data into training and test sets: ]
[2025-01-29 14:30:02,633: INFO: data_transformation: (1199, 12): ]
[2025-01-29 14:30:02,633: INFO: data_transformation: (400, 12): ]
[2025-01-29 14:30:02,635: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x: ]
[2025-01-29 14:30:02,636: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<: ]
[2025-01-29 14:30:02,636: ERROR: main: 'ModelTrainerTrainingPipeline' object has no attribute 'initate_model_trainin': ]
Traceback (most recent call last):
  File "D:\DS\project\datascienceproject\main.py", line 40, in <module>
    data_ingestion.initate_model_trainin()
AttributeError: 'ModelTrainerTrainingPipeline' object has no attribute 'initate_model_trainin'
[2025-01-29 14:31:28,723: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<: ]
[2025-01-29 14:31:28,731: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:31:28,735: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:31:28,744: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:31:28,745: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:31:28,747: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-29 14:31:28,747: INFO: data_ingestion: File already exists: ]
[2025-01-29 14:31:28,751: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x: ]
[2025-01-29 14:31:28,758: INFO: main: >>>>>> stage Data Validation stage started <<<<<<: ]
[2025-01-29 14:31:28,774: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:31:28,776: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:31:28,782: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:31:28,783: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:31:28,784: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-29 14:31:28,827: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x: ]
[2025-01-29 14:31:28,828: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<: ]
[2025-01-29 14:31:28,834: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:31:28,838: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:31:28,845: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:31:28,846: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:31:28,847: INFO: common: created directory at: artifacts/data_transformation: ]
[2025-01-29 14:31:28,876: INFO: data_transformation: Splited data into training and test sets: ]
[2025-01-29 14:31:28,877: INFO: data_transformation: (1199, 12): ]
[2025-01-29 14:31:28,878: INFO: data_transformation: (400, 12): ]
[2025-01-29 14:31:28,879: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x: ]
[2025-01-29 14:31:28,880: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<: ]
[2025-01-29 14:31:28,883: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:31:28,886: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:31:28,893: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:31:28,894: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:31:28,894: ERROR: main: "'ConfigBox' object has no attribute 'elasticnet'": ]
Traceback (most recent call last):
  File "box\\box.py", line 594, in box.box.Box.__getitem__
KeyError: 'ElasticNet'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\box.py", line 633, in box.box.Box.__getattr__
  File "box\\box.py", line 621, in box.box.Box.__getitem__
box.exceptions.BoxKeyError: "'ElasticNet'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 635, in box.box.Box.__getattr__
AttributeError: 'ConfigBox' object has no attribute 'ElasticNet'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\config_box.py", line 29, in box.config_box.ConfigBox.__getattr__
  File "box\\box.py", line 649, in box.box.Box.__getattr__
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'ElasticNet'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 594, in box.box.Box.__getitem__
KeyError: 'elasticnet'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\box.py", line 633, in box.box.Box.__getattr__
  File "box\\box.py", line 621, in box.box.Box.__getitem__
box.exceptions.BoxKeyError: "'elasticnet'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 635, in box.box.Box.__getattr__
AttributeError: 'ConfigBox' object has no attribute 'elasticnet'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\DS\project\datascienceproject\main.py", line 40, in <module>
    data_ingestion.initiate_model_training()
  File "D:\DS\project\datascienceproject\src\datascienceproject\pipeline\model_trainer_pipeline.py", line 13, in initiate_model_training
    model_trainer_config = config.get_model_trainer_config()
  File "D:\DS\project\datascienceproject\src\datascienceproject\config\configuration.py", line 56, in get_model_trainer_config
    params = self.params.ElasticNet
  File "box\\config_box.py", line 31, in box.config_box.ConfigBox.__getattr__
  File "box\\box.py", line 649, in box.box.Box.__getattr__
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'elasticnet'"
[2025-01-29 14:34:31,128: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<: ]
[2025-01-29 14:34:31,133: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:34:31,134: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:34:31,138: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:34:31,139: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:34:31,141: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-29 14:34:31,142: INFO: data_ingestion: File already exists: ]
[2025-01-29 14:34:31,150: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x: ]
[2025-01-29 14:34:31,156: INFO: main: >>>>>> stage Data Validation stage started <<<<<<: ]
[2025-01-29 14:34:31,163: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:34:31,165: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:34:31,168: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:34:31,169: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:34:31,170: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-29 14:34:31,207: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x: ]
[2025-01-29 14:34:31,208: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<: ]
[2025-01-29 14:34:31,215: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:34:31,217: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:34:31,219: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:34:31,222: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:34:31,223: INFO: common: created directory at: artifacts/data_transformation: ]
[2025-01-29 14:34:31,253: INFO: data_transformation: Splited data into training and test sets: ]
[2025-01-29 14:34:31,257: INFO: data_transformation: (1199, 12): ]
[2025-01-29 14:34:31,258: INFO: data_transformation: (400, 12): ]
[2025-01-29 14:34:31,260: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x: ]
[2025-01-29 14:34:31,261: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<: ]
[2025-01-29 14:34:31,265: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:34:31,266: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:34:31,269: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:34:31,278: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:34:31,279: ERROR: main: "'ConfigBox' object has no attribute 'elasticnet'": ]
Traceback (most recent call last):
  File "box\\box.py", line 594, in box.box.Box.__getitem__
KeyError: 'ElasticNet'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\box.py", line 633, in box.box.Box.__getattr__
  File "box\\box.py", line 621, in box.box.Box.__getitem__
box.exceptions.BoxKeyError: "'ElasticNet'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 635, in box.box.Box.__getattr__
AttributeError: 'ConfigBox' object has no attribute 'ElasticNet'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\config_box.py", line 29, in box.config_box.ConfigBox.__getattr__
  File "box\\box.py", line 649, in box.box.Box.__getattr__
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'ElasticNet'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 594, in box.box.Box.__getitem__
KeyError: 'elasticnet'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\box.py", line 633, in box.box.Box.__getattr__
  File "box\\box.py", line 621, in box.box.Box.__getitem__
box.exceptions.BoxKeyError: "'elasticnet'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 635, in box.box.Box.__getattr__
AttributeError: 'ConfigBox' object has no attribute 'elasticnet'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\DS\project\datascienceproject\main.py", line 40, in <module>
    data_ingestion.initiate_model_training()
  File "D:\DS\project\datascienceproject\src\datascienceproject\pipeline\model_trainer_pipeline.py", line 13, in initiate_model_training
    model_trainer_config = config.get_model_trainer_config()
  File "D:\DS\project\datascienceproject\src\datascienceproject\config\configuration.py", line 56, in get_model_trainer_config
    params = self.params.ElasticNet
  File "box\\config_box.py", line 31, in box.config_box.ConfigBox.__getattr__
  File "box\\box.py", line 649, in box.box.Box.__getattr__
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'elasticnet'"
[2025-01-29 14:36:54,370: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<: ]
[2025-01-29 14:36:54,375: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:36:54,389: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:36:54,393: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:36:54,394: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:36:54,395: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-29 14:36:54,396: INFO: data_ingestion: File already exists: ]
[2025-01-29 14:36:54,402: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x: ]
[2025-01-29 14:36:54,404: INFO: main: >>>>>> stage Data Validation stage started <<<<<<: ]
[2025-01-29 14:36:54,418: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:36:54,420: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:36:54,433: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:36:54,435: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:36:54,436: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-29 14:36:54,472: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x: ]
[2025-01-29 14:36:54,473: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<: ]
[2025-01-29 14:36:54,481: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:36:54,483: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:36:54,491: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:36:54,492: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:36:54,493: INFO: common: created directory at: artifacts/data_transformation: ]
[2025-01-29 14:36:54,536: INFO: data_transformation: Splited data into training and test sets: ]
[2025-01-29 14:36:54,537: INFO: data_transformation: (1199, 12): ]
[2025-01-29 14:36:54,538: INFO: data_transformation: (400, 12): ]
[2025-01-29 14:36:54,539: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x: ]
[2025-01-29 14:36:54,540: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<: ]
[2025-01-29 14:36:54,551: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:36:54,552: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:36:54,556: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:36:54,558: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:36:54,558: ERROR: main: "'ConfigBox' object has no attribute 'elastic_net'": ]
Traceback (most recent call last):
  File "box\\box.py", line 594, in box.box.Box.__getitem__
KeyError: 'Elastic_Net'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\box.py", line 633, in box.box.Box.__getattr__
  File "box\\box.py", line 621, in box.box.Box.__getitem__
box.exceptions.BoxKeyError: "'Elastic_Net'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 635, in box.box.Box.__getattr__
AttributeError: 'ConfigBox' object has no attribute 'Elastic_Net'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\config_box.py", line 29, in box.config_box.ConfigBox.__getattr__
  File "box\\box.py", line 649, in box.box.Box.__getattr__
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'Elastic_Net'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 594, in box.box.Box.__getitem__
KeyError: 'elastic_net'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\box.py", line 633, in box.box.Box.__getattr__
  File "box\\box.py", line 621, in box.box.Box.__getitem__
box.exceptions.BoxKeyError: "'elastic_net'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 635, in box.box.Box.__getattr__
AttributeError: 'ConfigBox' object has no attribute 'elastic_net'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\DS\project\datascienceproject\main.py", line 40, in <module>
    data_ingestion.initiate_model_training()
  File "D:\DS\project\datascienceproject\src\datascienceproject\pipeline\model_trainer_pipeline.py", line 13, in initiate_model_training
    model_trainer_config = config.get_model_trainer_config()
  File "D:\DS\project\datascienceproject\src\datascienceproject\config\configuration.py", line 56, in get_model_trainer_config
    params = self.params.Elastic_Net
  File "box\\config_box.py", line 31, in box.config_box.ConfigBox.__getattr__
  File "box\\box.py", line 649, in box.box.Box.__getattr__
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'elastic_net'"
[2025-01-29 14:38:16,597: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<: ]
[2025-01-29 14:38:16,601: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:38:16,604: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:38:16,607: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:38:16,608: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:38:16,609: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-29 14:38:16,609: INFO: data_ingestion: File already exists: ]
[2025-01-29 14:38:16,613: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x: ]
[2025-01-29 14:38:16,613: INFO: main: >>>>>> stage Data Validation stage started <<<<<<: ]
[2025-01-29 14:38:16,619: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:38:16,620: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:38:16,626: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:38:16,628: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:38:16,629: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-29 14:38:16,658: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x: ]
[2025-01-29 14:38:16,659: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<: ]
[2025-01-29 14:38:16,672: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:38:16,674: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:38:16,679: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:38:16,681: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:38:16,682: INFO: common: created directory at: artifacts/data_transformation: ]
[2025-01-29 14:38:16,715: INFO: data_transformation: Splited data into training and test sets: ]
[2025-01-29 14:38:16,716: INFO: data_transformation: (1199, 12): ]
[2025-01-29 14:38:16,717: INFO: data_transformation: (400, 12): ]
[2025-01-29 14:38:16,720: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x: ]
[2025-01-29 14:38:16,721: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<: ]
[2025-01-29 14:38:16,728: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:38:16,730: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:38:16,733: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:38:16,734: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:38:16,736: INFO: common: created directory at: artifacts/model_trainer: ]
[2025-01-29 14:38:16,738: ERROR: main: ModelTrainerConfig.__init__() got an unexpected keyword argument 'alpha': ]
Traceback (most recent call last):
  File "D:\DS\project\datascienceproject\main.py", line 40, in <module>
    data_ingestion.initiate_model_training()
  File "D:\DS\project\datascienceproject\src\datascienceproject\pipeline\model_trainer_pipeline.py", line 13, in initiate_model_training
    model_trainer_config = config.get_model_trainer_config()
  File "D:\DS\project\datascienceproject\src\datascienceproject\config\configuration.py", line 61, in get_model_trainer_config
    model_trainer_config = ModelTrainerConfig(
TypeError: ModelTrainerConfig.__init__() got an unexpected keyword argument 'alpha'
[2025-01-29 14:39:10,796: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<: ]
[2025-01-29 14:39:10,802: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:39:10,803: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:39:10,806: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:39:10,807: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:39:10,807: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-29 14:39:10,809: INFO: data_ingestion: File already exists: ]
[2025-01-29 14:39:10,814: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x: ]
[2025-01-29 14:39:10,815: INFO: main: >>>>>> stage Data Validation stage started <<<<<<: ]
[2025-01-29 14:39:10,818: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:39:10,821: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:39:10,826: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:39:10,827: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:39:10,829: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-29 14:39:10,861: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x: ]
[2025-01-29 14:39:10,865: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<: ]
[2025-01-29 14:39:10,875: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:39:10,876: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:39:10,880: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:39:10,883: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:39:10,883: INFO: common: created directory at: artifacts/data_transformation: ]
[2025-01-29 14:39:10,916: INFO: data_transformation: Splited data into training and test sets: ]
[2025-01-29 14:39:10,918: INFO: data_transformation: (1199, 12): ]
[2025-01-29 14:39:10,918: INFO: data_transformation: (400, 12): ]
[2025-01-29 14:39:10,919: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x: ]
[2025-01-29 14:39:10,919: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<: ]
[2025-01-29 14:39:10,922: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:39:10,924: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:39:10,927: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:39:10,927: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:39:10,928: INFO: common: created directory at: artifacts/model_trainer: ]
[2025-01-29 14:39:10,929: ERROR: main: ModelTrainerConfig.__init__() got an unexpected keyword argument 'alpha': ]
Traceback (most recent call last):
  File "D:\DS\project\datascienceproject\main.py", line 40, in <module>
    data_ingestion.initiate_model_training()
  File "D:\DS\project\datascienceproject\src\datascienceproject\pipeline\model_trainer_pipeline.py", line 13, in initiate_model_training
    model_trainer_config = config.get_model_trainer_config()
  File "D:\DS\project\datascienceproject\src\datascienceproject\config\configuration.py", line 61, in get_model_trainer_config
    model_trainer_config = ModelTrainerConfig(
TypeError: ModelTrainerConfig.__init__() got an unexpected keyword argument 'alpha'
[2025-01-29 14:39:59,911: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<: ]
[2025-01-29 14:39:59,915: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:39:59,916: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:39:59,921: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:39:59,922: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:39:59,922: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-29 14:39:59,923: INFO: data_ingestion: File already exists: ]
[2025-01-29 14:39:59,930: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x: ]
[2025-01-29 14:39:59,930: INFO: main: >>>>>> stage Data Validation stage started <<<<<<: ]
[2025-01-29 14:39:59,937: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:39:59,938: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:39:59,943: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:39:59,944: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:39:59,946: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-29 14:39:59,975: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x: ]
[2025-01-29 14:39:59,976: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<: ]
[2025-01-29 14:39:59,980: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:39:59,982: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:39:59,987: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:39:59,988: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:39:59,989: INFO: common: created directory at: artifacts/data_transformation: ]
[2025-01-29 14:40:00,045: INFO: data_transformation: Splited data into training and test sets: ]
[2025-01-29 14:40:00,057: INFO: data_transformation: (1199, 12): ]
[2025-01-29 14:40:00,063: INFO: data_transformation: (400, 12): ]
[2025-01-29 14:40:00,076: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x: ]
[2025-01-29 14:40:00,077: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<: ]
[2025-01-29 14:40:00,087: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:40:00,090: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:40:00,096: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:40:00,098: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:40:00,105: INFO: common: created directory at: artifacts/model_trainer: ]
[2025-01-29 14:40:00,106: ERROR: main: ModelTrainerConfig.__init__() got an unexpected keyword argument 'target_column': ]
Traceback (most recent call last):
  File "D:\DS\project\datascienceproject\main.py", line 40, in <module>
    data_ingestion.initiate_model_training()
  File "D:\DS\project\datascienceproject\src\datascienceproject\pipeline\model_trainer_pipeline.py", line 13, in initiate_model_training
    model_trainer_config = config.get_model_trainer_config()
  File "D:\DS\project\datascienceproject\src\datascienceproject\config\configuration.py", line 61, in get_model_trainer_config
    model_trainer_config = ModelTrainerConfig(
TypeError: ModelTrainerConfig.__init__() got an unexpected keyword argument 'target_column'
[2025-01-29 14:40:37,290: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<: ]
[2025-01-29 14:40:37,295: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:40:37,296: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:40:37,300: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:40:37,301: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:40:37,302: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-29 14:40:37,302: INFO: data_ingestion: File already exists: ]
[2025-01-29 14:40:37,308: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x: ]
[2025-01-29 14:40:37,312: INFO: main: >>>>>> stage Data Validation stage started <<<<<<: ]
[2025-01-29 14:40:37,315: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:40:37,317: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:40:37,320: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:40:37,320: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:40:37,321: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-29 14:40:37,357: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x: ]
[2025-01-29 14:40:37,363: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<: ]
[2025-01-29 14:40:37,369: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:40:37,371: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:40:37,379: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:40:37,380: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:40:37,381: INFO: common: created directory at: artifacts/data_transformation: ]
[2025-01-29 14:40:37,418: INFO: data_transformation: Splited data into training and test sets: ]
[2025-01-29 14:40:37,419: INFO: data_transformation: (1199, 12): ]
[2025-01-29 14:40:37,420: INFO: data_transformation: (400, 12): ]
[2025-01-29 14:40:37,421: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x: ]
[2025-01-29 14:40:37,422: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<: ]
[2025-01-29 14:40:37,430: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 14:40:37,432: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 14:40:37,435: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 14:40:37,436: INFO: common: created directory at: artifacts: ]
[2025-01-29 14:40:37,437: INFO: common: created directory at: artifacts/model_trainer: ]
[2025-01-29 14:40:37,696: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x: ]
[2025-01-29 20:40:37,323: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<: ]
[2025-01-29 20:40:37,349: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 20:40:37,389: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 20:40:37,401: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 20:40:37,404: INFO: common: created directory at: artifacts: ]
[2025-01-29 20:40:37,408: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-29 20:40:37,409: INFO: data_ingestion: File already exists: ]
[2025-01-29 20:40:37,489: INFO: main: >>>>>> stage Model evaluation stage completed <<<<<<

x==========x: ]
[2025-01-29 20:40:37,490: INFO: main: >>>>>> stage Data Validation stage started <<<<<<: ]
[2025-01-29 20:40:37,496: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 20:40:37,498: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 20:40:37,500: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 20:40:37,501: INFO: common: created directory at: artifacts: ]
[2025-01-29 20:40:37,514: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-29 20:40:37,663: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x: ]
[2025-01-29 20:40:37,666: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<: ]
[2025-01-29 20:40:37,689: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 20:40:37,691: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 20:40:37,693: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 20:40:37,693: INFO: common: created directory at: artifacts: ]
[2025-01-29 20:40:37,694: INFO: common: created directory at: artifacts/data_transformation: ]
[2025-01-29 20:40:37,922: INFO: data_transformation: Splited data into training and test sets: ]
[2025-01-29 20:40:37,923: INFO: data_transformation: (1199, 12): ]
[2025-01-29 20:40:37,923: INFO: data_transformation: (400, 12): ]
[2025-01-29 20:40:37,932: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x: ]
[2025-01-29 20:40:37,933: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<: ]
[2025-01-29 20:40:37,938: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 20:40:37,941: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 20:40:37,946: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 20:40:37,947: INFO: common: created directory at: artifacts: ]
[2025-01-29 20:40:37,949: INFO: common: created directory at: artifacts/model_trainer: ]
[2025-01-29 20:40:38,112: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x: ]
[2025-01-29 20:40:38,113: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<: ]
[2025-01-29 20:40:38,121: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 20:40:38,123: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 20:40:38,128: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 20:40:38,129: INFO: common: created directory at: artifacts: ]
[2025-01-29 20:40:38,130: ERROR: main: "'ConfigBox' object has no attribute 'elasticnet'": ]
Traceback (most recent call last):
  File "box\\box.py", line 594, in box.box.Box.__getitem__
KeyError: 'ElasticNet'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\box.py", line 633, in box.box.Box.__getattr__
  File "box\\box.py", line 621, in box.box.Box.__getitem__
box.exceptions.BoxKeyError: "'ElasticNet'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 635, in box.box.Box.__getattr__
AttributeError: 'ConfigBox' object has no attribute 'ElasticNet'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\config_box.py", line 29, in box.config_box.ConfigBox.__getattr__
  File "box\\box.py", line 649, in box.box.Box.__getattr__
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'ElasticNet'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 594, in box.box.Box.__getitem__
KeyError: 'elasticnet'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\box.py", line 633, in box.box.Box.__getattr__
  File "box\\box.py", line 621, in box.box.Box.__getitem__
box.exceptions.BoxKeyError: "'elasticnet'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 635, in box.box.Box.__getattr__
AttributeError: 'ConfigBox' object has no attribute 'elasticnet'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\DS\project\datascienceproject\main.py", line 51, in <module>
    data_ingestion.initiate_model_evaluation()
  File "D:\DS\project\datascienceproject\src\datascienceproject\pipeline\model_eval_pipeline.py", line 13, in initiate_model_evaluation
    model_evaluation_config = config.get_model_evaluation_config()
  File "D:\DS\project\datascienceproject\src\datascienceproject\config\configuration.py", line 75, in get_model_evaluation_config
    params=self.params.ElasticNet
  File "box\\config_box.py", line 31, in box.config_box.ConfigBox.__getattr__
  File "box\\box.py", line 649, in box.box.Box.__getattr__
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'elasticnet'"
[2025-01-29 20:44:03,305: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<: ]
[2025-01-29 20:44:03,326: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 20:44:03,349: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 20:44:03,352: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 20:44:03,360: INFO: common: created directory at: artifacts: ]
[2025-01-29 20:44:03,361: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-29 20:44:03,364: INFO: data_ingestion: File already exists: ]
[2025-01-29 20:44:03,413: INFO: main: >>>>>> stage Model evaluation stage completed <<<<<<

x==========x: ]
[2025-01-29 20:44:03,413: INFO: main: >>>>>> stage Data Validation stage started <<<<<<: ]
[2025-01-29 20:44:03,417: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 20:44:03,418: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 20:44:03,421: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 20:44:03,422: INFO: common: created directory at: artifacts: ]
[2025-01-29 20:44:03,423: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-29 20:44:03,519: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x: ]
[2025-01-29 20:44:03,520: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<: ]
[2025-01-29 20:44:03,528: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 20:44:03,532: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 20:44:03,536: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 20:44:03,537: INFO: common: created directory at: artifacts: ]
[2025-01-29 20:44:03,538: INFO: common: created directory at: artifacts/data_transformation: ]
[2025-01-29 20:44:03,682: INFO: data_transformation: Splited data into training and test sets: ]
[2025-01-29 20:44:03,685: INFO: data_transformation: (1199, 12): ]
[2025-01-29 20:44:03,687: INFO: data_transformation: (400, 12): ]
[2025-01-29 20:44:03,700: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x: ]
[2025-01-29 20:44:03,701: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<: ]
[2025-01-29 20:44:03,707: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 20:44:03,711: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 20:44:03,716: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 20:44:03,718: INFO: common: created directory at: artifacts: ]
[2025-01-29 20:44:03,718: INFO: common: created directory at: artifacts/model_trainer: ]
[2025-01-29 20:44:03,884: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x: ]
[2025-01-29 20:44:03,885: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<: ]
[2025-01-29 20:44:03,912: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 20:44:03,916: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 20:44:03,921: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 20:44:03,922: INFO: common: created directory at: artifacts: ]
[2025-01-29 20:44:03,923: ERROR: main: "'ConfigBox' object has no attribute 'elasticnet'": ]
Traceback (most recent call last):
  File "box\\box.py", line 594, in box.box.Box.__getitem__
KeyError: 'ElasticNet'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\box.py", line 633, in box.box.Box.__getattr__
  File "box\\box.py", line 621, in box.box.Box.__getitem__
box.exceptions.BoxKeyError: "'ElasticNet'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 635, in box.box.Box.__getattr__
AttributeError: 'ConfigBox' object has no attribute 'ElasticNet'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\config_box.py", line 29, in box.config_box.ConfigBox.__getattr__
  File "box\\box.py", line 649, in box.box.Box.__getattr__
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'ElasticNet'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 594, in box.box.Box.__getitem__
KeyError: 'elasticnet'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\box.py", line 633, in box.box.Box.__getattr__
  File "box\\box.py", line 621, in box.box.Box.__getitem__
box.exceptions.BoxKeyError: "'elasticnet'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 635, in box.box.Box.__getattr__
AttributeError: 'ConfigBox' object has no attribute 'elasticnet'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\DS\project\datascienceproject\main.py", line 51, in <module>
    data_ingestion.initiate_model_evaluation()
  File "D:\DS\project\datascienceproject\src\datascienceproject\pipeline\model_eval_pipeline.py", line 13, in initiate_model_evaluation
    model_evaluation_config = config.get_model_evaluation_config()
  File "D:\DS\project\datascienceproject\src\datascienceproject\config\configuration.py", line 75, in get_model_evaluation_config
    params=self.params.Elastic_net
  File "box\\config_box.py", line 31, in box.config_box.ConfigBox.__getattr__
  File "box\\box.py", line 649, in box.box.Box.__getattr__
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'elasticnet'"
[2025-01-29 20:44:13,649: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<: ]
[2025-01-29 20:44:13,654: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 20:44:13,656: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 20:44:13,659: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 20:44:13,660: INFO: common: created directory at: artifacts: ]
[2025-01-29 20:44:13,661: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-29 20:44:13,661: INFO: data_ingestion: File already exists: ]
[2025-01-29 20:44:13,665: INFO: main: >>>>>> stage Model evaluation stage completed <<<<<<

x==========x: ]
[2025-01-29 20:44:13,666: INFO: main: >>>>>> stage Data Validation stage started <<<<<<: ]
[2025-01-29 20:44:13,673: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 20:44:13,674: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 20:44:13,726: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 20:44:13,737: INFO: common: created directory at: artifacts: ]
[2025-01-29 20:44:13,766: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-29 20:44:13,825: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x: ]
[2025-01-29 20:44:13,827: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<: ]
[2025-01-29 20:44:13,838: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 20:44:13,863: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 20:44:13,886: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 20:44:13,927: INFO: common: created directory at: artifacts: ]
[2025-01-29 20:44:13,947: INFO: common: created directory at: artifacts/data_transformation: ]
[2025-01-29 20:44:14,052: INFO: data_transformation: Splited data into training and test sets: ]
[2025-01-29 20:44:14,072: INFO: data_transformation: (1199, 12): ]
[2025-01-29 20:44:14,075: INFO: data_transformation: (400, 12): ]
[2025-01-29 20:44:14,077: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x: ]
[2025-01-29 20:44:14,080: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<: ]
[2025-01-29 20:44:14,089: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 20:44:14,091: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 20:44:14,095: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 20:44:14,096: INFO: common: created directory at: artifacts: ]
[2025-01-29 20:44:14,097: INFO: common: created directory at: artifacts/model_trainer: ]
[2025-01-29 20:44:14,145: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x: ]
[2025-01-29 20:44:14,145: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<: ]
[2025-01-29 20:44:14,150: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 20:44:14,151: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 20:44:14,159: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 20:44:14,159: INFO: common: created directory at: artifacts: ]
[2025-01-29 20:44:14,160: INFO: common: created directory at: artifacts/model_evaluation: ]
[2025-01-29 20:44:18,106: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json: ]
[2025-01-29 20:44:54,319: INFO: main: >>>>>> stage Model evaluation stage completed <<<<<<

x==========x: ]
[2025-01-29 21:01:58,000: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<: ]
[2025-01-29 21:01:58,024: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 21:01:58,044: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 21:01:58,061: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 21:01:58,069: INFO: common: created directory at: artifacts: ]
[2025-01-29 21:01:58,070: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-29 21:01:58,070: INFO: data_ingestion: File already exists: ]
[2025-01-29 21:01:58,125: INFO: main: >>>>>> stage Model evaluation stage completed <<<<<<

x==========x: ]
[2025-01-29 21:01:58,126: INFO: main: >>>>>> stage Data Validation stage started <<<<<<: ]
[2025-01-29 21:01:58,132: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 21:01:58,133: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 21:01:58,137: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 21:01:58,138: INFO: common: created directory at: artifacts: ]
[2025-01-29 21:01:58,139: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-29 21:01:58,288: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x: ]
[2025-01-29 21:01:58,290: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<: ]
[2025-01-29 21:01:58,302: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 21:01:58,305: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 21:01:58,312: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 21:01:58,313: INFO: common: created directory at: artifacts: ]
[2025-01-29 21:01:58,314: INFO: common: created directory at: artifacts/data_transformation: ]
[2025-01-29 21:01:58,474: INFO: data_transformation: Splited data into training and test sets: ]
[2025-01-29 21:01:58,475: INFO: data_transformation: (1199, 12): ]
[2025-01-29 21:01:58,475: INFO: data_transformation: (400, 12): ]
[2025-01-29 21:01:58,487: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x: ]
[2025-01-29 21:01:58,488: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<: ]
[2025-01-29 21:01:58,494: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 21:01:58,496: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 21:01:58,498: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 21:01:58,499: INFO: common: created directory at: artifacts: ]
[2025-01-29 21:01:58,500: INFO: common: created directory at: artifacts/model_trainer: ]
[2025-01-29 21:01:58,711: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x: ]
[2025-01-29 21:01:58,712: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<: ]
[2025-01-29 21:01:58,718: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 21:01:58,720: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 21:01:58,724: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 21:01:58,725: INFO: common: created directory at: artifacts: ]
[2025-01-29 21:01:58,729: INFO: common: created directory at: artifacts/model_evaluation: ]
[2025-01-29 21:02:02,587: ERROR: main: API request to endpoint /api/2.0/mlflow/runs/create failed with error code 403 != 200. Response body: '': ]
Traceback (most recent call last):
  File "D:\DS\project\datascienceproject\main.py", line 51, in <module>
    data_ingestion.initiate_model_evaluation()
  File "D:\DS\project\datascienceproject\src\datascienceproject\pipeline\model_eval_pipeline.py", line 15, in initiate_model_evaluation
    model_evaluation.log_into_mlflow()
  File "D:\DS\project\datascienceproject\src\datascienceproject\components\model_evaluation.py", line 37, in log_into_mlflow
    with mlflow.start_run():
  File "D:\DS\project\datascienceproject\venv\lib\site-packages\mlflow\tracking\fluent.py", line 446, in start_run
    active_run_obj = client.create_run(
  File "D:\DS\project\datascienceproject\venv\lib\site-packages\mlflow\tracking\client.py", line 393, in create_run
    return self._tracking_client.create_run(experiment_id, start_time, tags, run_name)
  File "D:\DS\project\datascienceproject\venv\lib\site-packages\mlflow\tracking\_tracking_service\client.py", line 169, in create_run
    return self.store.create_run(
  File "D:\DS\project\datascienceproject\venv\lib\site-packages\mlflow\store\tracking\rest_store.py", line 212, in create_run
    response_proto = self._call_endpoint(CreateRun, req_body)
  File "D:\DS\project\datascienceproject\venv\lib\site-packages\mlflow\store\tracking\rest_store.py", line 82, in _call_endpoint
    return call_endpoint(self.get_host_creds(), endpoint, method, json_body, response_proto)
  File "D:\DS\project\datascienceproject\venv\lib\site-packages\mlflow\utils\rest_utils.py", line 379, in call_endpoint
    response = verify_rest_response(response, endpoint)
  File "D:\DS\project\datascienceproject\venv\lib\site-packages\mlflow\utils\rest_utils.py", line 255, in verify_rest_response
    raise MlflowException(
mlflow.exceptions.MlflowException: API request to endpoint /api/2.0/mlflow/runs/create failed with error code 403 != 200. Response body: ''
[2025-01-29 21:03:58,936: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<: ]
[2025-01-29 21:03:58,958: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 21:03:58,990: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 21:03:58,998: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 21:03:59,004: INFO: common: created directory at: artifacts: ]
[2025-01-29 21:03:59,005: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-29 21:03:59,005: INFO: data_ingestion: File already exists: ]
[2025-01-29 21:03:59,046: INFO: main: >>>>>> stage Model evaluation stage completed <<<<<<

x==========x: ]
[2025-01-29 21:03:59,047: INFO: main: >>>>>> stage Data Validation stage started <<<<<<: ]
[2025-01-29 21:03:59,051: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 21:03:59,052: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 21:03:59,055: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 21:03:59,057: INFO: common: created directory at: artifacts: ]
[2025-01-29 21:03:59,058: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-29 21:03:59,184: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x: ]
[2025-01-29 21:03:59,185: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<: ]
[2025-01-29 21:03:59,192: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 21:03:59,194: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 21:03:59,197: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 21:03:59,199: INFO: common: created directory at: artifacts: ]
[2025-01-29 21:03:59,200: INFO: common: created directory at: artifacts/data_transformation: ]
[2025-01-29 21:03:59,367: INFO: data_transformation: Splited data into training and test sets: ]
[2025-01-29 21:03:59,368: INFO: data_transformation: (1199, 12): ]
[2025-01-29 21:03:59,368: INFO: data_transformation: (400, 12): ]
[2025-01-29 21:03:59,378: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x: ]
[2025-01-29 21:03:59,378: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<: ]
[2025-01-29 21:03:59,384: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 21:03:59,386: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 21:03:59,388: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 21:03:59,389: INFO: common: created directory at: artifacts: ]
[2025-01-29 21:03:59,390: INFO: common: created directory at: artifacts/model_trainer: ]
[2025-01-29 21:03:59,572: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x: ]
[2025-01-29 21:03:59,577: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<: ]
[2025-01-29 21:03:59,585: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 21:03:59,587: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 21:03:59,590: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 21:03:59,592: INFO: common: created directory at: artifacts: ]
[2025-01-29 21:03:59,594: INFO: common: created directory at: artifacts/model_evaluation: ]
[2025-01-29 21:04:12,432: WARNING: connectionpool: Retrying (Retry(total=6, connect=7, read=6, redirect=7, status=7)) after connection broken by 'RemoteDisconnected('Remote end closed connection without response')': /Devampatel94/datascienceproject.mlflow/api/2.0/mlflow/runs/create: ]
[2025-01-29 21:04:12,954: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json: ]
[2025-01-29 21:04:42,366: INFO: main: >>>>>> stage Model evaluation stage completed <<<<<<

x==========x: ]
[2025-01-29 21:33:16,880: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://172.31.79.118:8080: ]
[2025-01-29 21:33:16,894: INFO: _internal: [33mPress CTRL+C to quit[0m: ]
[2025-01-29 21:33:28,712: INFO: _internal: 172.31.79.118 - - [29/Jan/2025 21:33:28] "GET / HTTP/1.1" 200 -: ]
[2025-01-29 21:33:28,847: INFO: _internal: 172.31.79.118 - - [29/Jan/2025 21:33:28] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -: ]
[2025-01-29 21:33:28,936: INFO: _internal: 172.31.79.118 - - [29/Jan/2025 21:33:28] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -: ]
[2025-01-29 21:33:51,153: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<: ]
[2025-01-29 21:33:51,174: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 21:33:51,218: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 21:33:51,224: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 21:33:51,229: INFO: common: created directory at: artifacts: ]
[2025-01-29 21:33:51,238: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-29 21:33:51,239: INFO: data_ingestion: File already exists: ]
[2025-01-29 21:33:51,293: INFO: main: >>>>>> stage Model evaluation stage completed <<<<<<

x==========x: ]
[2025-01-29 21:33:51,294: INFO: main: >>>>>> stage Data Validation stage started <<<<<<: ]
[2025-01-29 21:33:51,302: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 21:33:51,305: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 21:33:51,310: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 21:33:51,311: INFO: common: created directory at: artifacts: ]
[2025-01-29 21:33:51,318: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-29 21:33:51,426: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x: ]
[2025-01-29 21:33:51,428: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<: ]
[2025-01-29 21:33:51,435: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 21:33:51,437: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 21:33:51,440: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 21:33:51,441: INFO: common: created directory at: artifacts: ]
[2025-01-29 21:33:51,442: INFO: common: created directory at: artifacts/data_transformation: ]
[2025-01-29 21:33:51,625: INFO: data_transformation: Splited data into training and test sets: ]
[2025-01-29 21:33:51,627: INFO: data_transformation: (1199, 12): ]
[2025-01-29 21:33:51,627: INFO: data_transformation: (400, 12): ]
[2025-01-29 21:33:51,629: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x: ]
[2025-01-29 21:33:51,629: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<: ]
[2025-01-29 21:33:51,633: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 21:33:51,635: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 21:33:51,639: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 21:33:51,640: INFO: common: created directory at: artifacts: ]
[2025-01-29 21:33:51,641: INFO: common: created directory at: artifacts/model_trainer: ]
[2025-01-29 21:33:51,783: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x: ]
[2025-01-29 21:33:51,784: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<: ]
[2025-01-29 21:33:51,788: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 21:33:51,790: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 21:33:51,793: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 21:33:51,793: INFO: common: created directory at: artifacts: ]
[2025-01-29 21:33:51,794: INFO: common: created directory at: artifacts/model_evaluation: ]
[2025-01-29 21:33:56,894: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json: ]
[2025-01-29 21:35:52,113: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://172.31.79.118:8080: ]
[2025-01-29 21:35:52,117: INFO: _internal: [33mPress CTRL+C to quit[0m: ]
[2025-01-29 21:35:57,947: INFO: _internal: 172.31.79.118 - - [29/Jan/2025 21:35:57] "GET / HTTP/1.1" 200 -: ]
[2025-01-29 21:35:58,079: INFO: _internal: 172.31.79.118 - - [29/Jan/2025 21:35:58] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -: ]
[2025-01-29 21:36:22,026: INFO: _internal: 172.31.79.118 - - [29/Jan/2025 21:36:22] "POST /predict HTTP/1.1" 200 -: ]
[2025-01-29 21:37:05,160: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<: ]
[2025-01-29 21:37:05,185: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 21:37:05,205: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 21:37:05,212: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 21:37:05,218: INFO: common: created directory at: artifacts: ]
[2025-01-29 21:37:05,219: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-29 21:37:05,220: INFO: data_ingestion: File already exists: ]
[2025-01-29 21:37:05,241: INFO: main: >>>>>> stage Model evaluation stage completed <<<<<<

x==========x: ]
[2025-01-29 21:37:05,244: INFO: main: >>>>>> stage Data Validation stage started <<<<<<: ]
[2025-01-29 21:37:05,251: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 21:37:05,253: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 21:37:05,256: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 21:37:05,258: INFO: common: created directory at: artifacts: ]
[2025-01-29 21:37:05,258: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-29 21:37:05,358: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x: ]
[2025-01-29 21:37:05,361: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<: ]
[2025-01-29 21:37:05,383: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 21:37:05,384: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 21:37:05,386: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 21:37:05,387: INFO: common: created directory at: artifacts: ]
[2025-01-29 21:37:05,388: INFO: common: created directory at: artifacts/data_transformation: ]
[2025-01-29 21:37:05,548: INFO: data_transformation: Splited data into training and test sets: ]
[2025-01-29 21:37:05,548: INFO: data_transformation: (1199, 12): ]
[2025-01-29 21:37:05,549: INFO: data_transformation: (400, 12): ]
[2025-01-29 21:37:05,559: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x: ]
[2025-01-29 21:37:05,560: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<: ]
[2025-01-29 21:37:05,570: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 21:37:05,572: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 21:37:05,574: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 21:37:05,575: INFO: common: created directory at: artifacts: ]
[2025-01-29 21:37:05,580: INFO: common: created directory at: artifacts/model_trainer: ]
[2025-01-29 21:37:05,733: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x: ]
[2025-01-29 21:37:05,734: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<: ]
[2025-01-29 21:37:05,739: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 21:37:05,742: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 21:37:05,747: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 21:37:05,749: INFO: common: created directory at: artifacts: ]
[2025-01-29 21:37:05,750: INFO: common: created directory at: artifacts/model_evaluation: ]
[2025-01-29 21:37:09,823: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json: ]
[2025-01-29 21:37:52,179: INFO: main: >>>>>> stage Model evaluation stage completed <<<<<<

x==========x: ]
[2025-01-29 21:37:52,717: INFO: _internal: 172.31.79.118 - - [29/Jan/2025 21:37:52] "GET /train HTTP/1.1" 200 -: ]
[2025-01-29 21:38:02,955: INFO: _internal: 172.31.79.118 - - [29/Jan/2025 21:38:02] "GET /predict HTTP/1.1" 200 -: ]
[2025-01-29 21:38:03,045: INFO: _internal: 172.31.79.118 - - [29/Jan/2025 21:38:03] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -: ]
[2025-01-29 21:38:20,282: INFO: _internal: 172.31.79.118 - - [29/Jan/2025 21:38:20] "POST /predict HTTP/1.1" 200 -: ]
[2025-01-29 21:40:59,061: INFO: _internal: 172.31.79.118 - - [29/Jan/2025 21:40:59] "GET /predict HTTP/1.1" 200 -: ]
[2025-01-29 21:41:00,513: INFO: _internal: 172.31.79.118 - - [29/Jan/2025 21:41:00] "POST /predict HTTP/1.1" 200 -: ]
[2025-01-29 21:46:57,788: INFO: _internal: 172.31.79.118 - - [29/Jan/2025 21:46:57] "POST /predict HTTP/1.1" 200 -: ]
[2025-01-29 21:46:59,233: INFO: _internal: 172.31.79.118 - - [29/Jan/2025 21:46:59] "GET /predict HTTP/1.1" 200 -: ]
[2025-01-29 21:47:26,899: INFO: _internal: 172.31.79.118 - - [29/Jan/2025 21:47:26] "GET / HTTP/1.1" 200 -: ]
[2025-01-29 21:47:27,071: INFO: _internal: 172.31.79.118 - - [29/Jan/2025 21:47:27] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -: ]
[2025-01-29 21:47:33,377: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<: ]
[2025-01-29 21:47:33,391: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 21:47:33,432: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 21:47:33,437: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 21:47:33,445: INFO: common: created directory at: artifacts: ]
[2025-01-29 21:47:33,446: INFO: common: created directory at: artifacts/data_ingestion: ]
[2025-01-29 21:47:33,447: INFO: data_ingestion: File already exists: ]
[2025-01-29 21:47:33,491: INFO: main: >>>>>> stage Model evaluation stage completed <<<<<<

x==========x: ]
[2025-01-29 21:47:33,493: INFO: main: >>>>>> stage Data Validation stage started <<<<<<: ]
[2025-01-29 21:47:33,500: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 21:47:33,503: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 21:47:33,509: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 21:47:33,510: INFO: common: created directory at: artifacts: ]
[2025-01-29 21:47:33,513: INFO: common: created directory at: artifacts/data_validation: ]
[2025-01-29 21:47:33,647: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x: ]
[2025-01-29 21:47:33,651: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<: ]
[2025-01-29 21:47:33,659: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 21:47:33,662: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 21:47:33,666: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 21:47:33,667: INFO: common: created directory at: artifacts: ]
[2025-01-29 21:47:33,669: INFO: common: created directory at: artifacts/data_transformation: ]
[2025-01-29 21:47:33,828: INFO: data_transformation: Splited data into training and test sets: ]
[2025-01-29 21:47:33,830: INFO: data_transformation: (1199, 12): ]
[2025-01-29 21:47:33,831: INFO: data_transformation: (400, 12): ]
[2025-01-29 21:47:33,852: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x: ]
[2025-01-29 21:47:33,854: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<: ]
[2025-01-29 21:47:33,863: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 21:47:33,866: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 21:47:33,871: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 21:47:33,874: INFO: common: created directory at: artifacts: ]
[2025-01-29 21:47:33,875: INFO: common: created directory at: artifacts/model_trainer: ]
[2025-01-29 21:47:34,108: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x: ]
[2025-01-29 21:47:34,109: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<: ]
[2025-01-29 21:47:34,122: INFO: common: yaml file: config\config.yaml loaded successfully: ]
[2025-01-29 21:47:34,125: INFO: common: yaml file: params.yaml loaded successfully: ]
[2025-01-29 21:47:34,128: INFO: common: yaml file: schema.yaml loaded successfully: ]
[2025-01-29 21:47:34,129: INFO: common: created directory at: artifacts: ]
[2025-01-29 21:47:34,136: INFO: common: created directory at: artifacts/model_evaluation: ]
[2025-01-29 21:47:38,320: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json: ]
[2025-01-29 21:48:05,279: INFO: main: >>>>>> stage Model evaluation stage completed <<<<<<

x==========x: ]
[2025-01-29 21:48:05,752: INFO: _internal: 172.31.79.118 - - [29/Jan/2025 21:48:05] "GET /train HTTP/1.1" 200 -: ]
[2025-01-29 21:48:11,022: INFO: _internal: 172.31.79.118 - - [29/Jan/2025 21:48:11] "GET / HTTP/1.1" 200 -: ]
[2025-01-29 21:48:11,085: INFO: _internal: 172.31.79.118 - - [29/Jan/2025 21:48:11] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -: ]
[2025-01-29 21:48:13,459: INFO: _internal: 172.31.79.118 - - [29/Jan/2025 21:48:13] "GET / HTTP/1.1" 200 -: ]
[2025-01-29 21:48:13,510: INFO: _internal: 172.31.79.118 - - [29/Jan/2025 21:48:13] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -: ]
[2025-01-29 21:49:48,341: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://172.31.79.118:8080: ]
[2025-01-29 21:49:48,342: INFO: _internal: [33mPress CTRL+C to quit[0m: ]
[2025-01-29 21:49:48,362: INFO: _internal:  * Restarting with stat: ]
[2025-01-29 21:49:49,569: WARNING: _internal:  * Debugger is active!: ]
[2025-01-29 21:49:49,573: INFO: _internal:  * Debugger PIN: 335-065-808: ]
[2025-01-29 21:52:03,326: INFO: _internal: 172.31.79.118 - - [29/Jan/2025 21:52:03] "GET / HTTP/1.1" 200 -: ]
[2025-01-29 21:52:22,207: INFO: _internal: 172.31.79.118 - - [29/Jan/2025 21:52:22] "POST /predict HTTP/1.1" 200 -: ]
